Automattic/jetpack
# CRM: remove built CSS from the monorepo, plus .gitignore and .gitattributes cleanup

## Proposed changes:

* This PR removes all built CRM CSS in the monorepo (CSS that is built via Webpack from SCSS files). The CSS files will still be built and added to the mirror repo (for production).
* The PR also updates the `.gitattributes` and `.gitignore` files, as well as removing the `.distignore` file which serves no obvious purpose now that CRM is built and deployed using monorepo tooling.

### Other information:


## Jetpack product discussion

https://github.com/Automattic/zero-bs-crm/issues/3056

## Does this pull request change what data or activity we track or use?

No.

## Testing instructions:

* To test this PR out, check out this branch in a local development environment.
* Run `jetpack build plugins/crm` - by doing so this should not add new untracked files, or files with changes needing committing.
* Run `jetpack build plugins/crm --production` - the results should be the same.
* Try adding a change to a SCSS file and run a build again. You should see the change in the CSS file locally, but the file won't show as a changed file needing commit (it will continue to show as a deleted file with this PR checked out).
